src/modules/map/components/LocationPopupCopyInput.vue

```diff
@@ -63,11 +64,14 @@ export default {
             event.target.select()
         },
         async copyValue() {
-            await navigator.clipboard.writeText(this.value)
-
-            // Change button text and start the reset timer.
-            this.buttonText = 'copy_success'
-            this.resetTimeout = setTimeout(this.resetButton, this.resetDelay)
+            try {
+                await navigator.clipboard.writeText(this.value)
+                // Change button text and start the reset timer.
+                this.buttonText = 'copy_success'
+                this.resetTimeout = setTimeout(this.resetButton, this.resetDelay)
+            } catch (error) {
+                log.error(`Failed to copy to clipboard: ${error}`)
```

Comment:
you could even pass the error as a secondary argument, in case this error is not a string but an object
```
log.error('Failed to copy to clipboard', error)
```
Otherwise it can sometimes flatten the object and output `Object: object` in the console, instead of flattening it into a string

Reply:
Done